packages/loot-core/src/server/accounts/transaction-rules.ts

```diff
@@ -494,11 +541,18 @@ export function conditionsToAQL(conditions, { recurDateBounds = 100 } = {}) {
         return apply(field, '$eq', true);
       case 'false':
         return apply(field, '$eq', false);
+      case 'and':
+        debugger;
```

Comment:
```suggestion
```

Reply:
Hm. Maybe I should add it to the linter rules....

Reply:
also, thanks